soccer/src/soccer/strategy/agent/position/offense.cpp

```diff
@@ -14,6 +14,10 @@ std::optional<RobotIntent> Offense::derived_get_task(RobotIntent intent) {
     return state_to_task(intent);
 }
 
+std::string Offense::return_current_state() {
+    return "Offense" + static_cast<int>(current_state_);
```

Comment:
Same problem as in Defense, the same fix should work. Note the clang-tidy warning present in Github. In the future, please check these warnings before requesting a review. I am happy to help you fix them if you don't know how.